Dockerfile

```diff
@@ -1,12 +1,22 @@
-ARG BASE_IMAGE=apache/airflow:2.7.2-python3.10
+ARG BASE_IMAGE=apache/airflow:2.9.3-python3.10
 FROM $BASE_IMAGE
 
 RUN pip install --upgrade pip
 
-RUN pip install apache-airflow[amazon,celery,snowflake]==2.7.2 --constraint "https://raw.githubusercontent.com/apache/airflow/constraints-2.7.2/constraints-3.10.txt"
+RUN pip install apache-airflow[amazon,celery,snowflake]==2.9.3 --constraint "https://raw.githubusercontent.com/apache/airflow/constraints-2.9.3/constraints-3.10.txt"
 
 COPY requirements-airflow.txt /tmp/requirements-airflow.txt
 RUN pip install --no-cache-dir -r /tmp/requirements-airflow.txt
 
+USER root
+RUN apt-get update
+RUN apt-get -y install git
+
+# change back to base image user
+USER 50000
+
+# Installed from git a version of py-orca that is compatible with the version of the airflow image
+RUN pip install -e git+https://github.com/Sage-Bionetworks-Workflows/py-orca@7c4781c4750ca8025deb1c6665730d7d46844fbc#egg=py-orca[all]
+
```

Comment:
We should probably update py-orca so we don't need to do this. @BWMac Are you aware of the implications of upgrading py-orca with this and what/if any additional testing we'll need to do?

Related PR: https://github.com/Sage-Bionetworks-Workflows/py-orca/pull/45

Reply:
With the testing that you did in the py-orca PR I don't expect any breaking changes here (basically we just need to make sure DAGs can run).